### DISCUSSION TITLE
Rubygems owners

### DISCUSSION THREAD

**Discussion by ioquatix** [kickstarter/rack-attack#615] [General]
Hi @grzuy do you mind adding me as an owner on Rubygems so I can make a new release for Rack 3 support?

Also, we should remove old maintainers if they are no longer doing release management, especially if they don't have MFA setup (on Rubygems.org).

Thanks!

**Comment by grzuy** [comment]
Hi @ioquatix,

Sure!
Just added you.

**Reply by ioquatix** [reply]
Thanks!
